src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -45,6 +47,15 @@ public Test()
                     solution = solution.WithProjectCompilationOptions(projectId, compilationOptions);
 
                     var (analyzerConfigSource, remainingOptions) = CodeFixVerifierHelper.ConvertOptionsToAnalyzerConfig(DefaultFileExt, Options);
+                    if (EditorConfig is object)
```

Comment:
Can we pass `EditorConfig` as an argument to `ConvertOptionsToAnalyzerConfig` and share code between VB and C#?

Reply:
Done!